src/framework/ui/iuiactionsmodule.h

```diff
@@ -35,6 +35,14 @@ class IUiActionsModule
     virtual ~IUiActionsModule() = default;
 
     virtual const UiActionList& actionsList() const = 0;
+    virtual async::Channel<UiAction> actionChanged() const
```

Comment:
Perhaps we should change this channel a bit to be able to send multiple actions at once:

async::Channel< **UiActionList** > action**s**Changed() const

That would prevent potential notification spam

Reply:
changed to list